docs/source/manual/file_formats/reference_file.rst

```diff
@@ -17,60 +17,111 @@ An example of the file is shown as follows.
 Date: Mon Sep  9 16:41:53 2024
```

Comment:
Indent is needed on LINE 17 and LINE 18 as well.

Reply:
OK, thanks.